_patterns/Typing Facilitator.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+---
+# Also Known As
+
+# Context
+
+Either
+
+- A quick impromptu mobbing session
+- Mobbing with non-developers
+- Introducing mobbing to beginners
+
+# Purpose
+
+- Get going, reduce friction
+- Lower anxiety levels related to typing in public
+- Demo Driving skills
+- Could be used to convert a meeting to a working meeting ("Is it ok if we just *did it* right now?")
+
```

Comment:
This is already covered by "Get going....", but provides a bit more context.  It may be too long to be read quickly.

- To onboard new member(s), an existing member can explicitly review code or give a demo while implicitly introducing team culture - show, not tell.

P.S.  This is exactly what happened today (Friday) at the International Code Retreat.  Instead of explicitly answering a new member's questions about how we'd worked the day before, I suggested (rather strongly) that the two previous day's participants take comfortable roles of talker and typer, then at a good event break, I passed the keyboard to a newbie, who seamlessly followed the lead (instead of asking 20 questions).